atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -302,22 +302,29 @@ private void schemaMutationUnlock(long perOperationNodeId) {
     }
 
     private boolean trySchemaMutationUnlockOnce(long perOperationNodeId) {
-        CASResult result;
         try {
-            result = clientPool.runWithRetry(client -> {
+            return clientPool.runWithRetry(client -> {
                 Column existingColumn = queryExistingLockColumn(client);
+                long existingLockId = getLockIdFromColumn(existingColumn);
+                if (existingLockId == GLOBAL_DDL_LOCK_CLEARED_ID) {
+                    return true;
+                }
 
-                Preconditions.checkState(isValidColumnForLockId(existingColumn, perOperationNodeId),
-                        "Trying to unlock unowned lock");
+                Preconditions.checkState(existingLockId == perOperationNodeId,
+                        "Trying to unlock unowned lock. Expected [%s], but got [%s]",
+                        perOperationNodeId, existingLockId);
 
                 List<Column> ourExpectedLock = ImmutableList.of(existingColumn);
                 Column clearedLock = lockColumnWithValue(GLOBAL_DDL_LOCK_CLEARED_VALUE);
-                return writeDdlLockWithCas(client, ourExpectedLock, clearedLock);
+                CASResult casResult = writeDdlLockWithCas(client, ourExpectedLock, clearedLock);
+                if (casResult.isSuccess()) {
+                    log.info("Successfully released schema mutation lock with id [{}]", existingLockId);
```

Comment:
We could but the log looks weird that way, since we log the "acquired message" inside `runWithRetry`. There is additional text wrapped around the log when logging from inside `runWithRetry` so this was needed for consistency.